tests/subsys/zbus/integration/src/main.c

```diff
@@ -170,13 +209,17 @@ ZTEST(integration, test_basic)
 	zassert_equal(count_core, 3, NULL);
 	zassert_equal(count_net, 3, NULL);
 	zassert_equal(count_peripheral, 3, NULL);
+	zassert_equal(count_net_log, 3, NULL);
+	zassert_equal(count_net, lm->count_net, NULL);
 
 	zassert_equal(pkt.total, 6, "result was %d", pkt.total);
+	zassert_equal(pkt.total, lm->pkt_total, NULL);
 }
 
 ZTEST(integration, test_channel_set_enable)
 {
 	struct action_msg start = {true};
+	struct net_log_msg *lm = (struct net_log_msg *)zbus_chan_const_msg(&net_log_chan);
```

Comment:
do you need the casting? could you declar lm as const and that's it?

Reply:
No casting is needed. Fixed.